insert_string_tpl.h

```diff
@@ -61,7 +61,7 @@ ZLIB_INTERNAL Pos QUICK_INSERT_STRING(deflate_state *const s, const uint32_t str
  *    (except for the last MIN_MATCH-1 bytes of the input file).
  */
 ZLIB_INTERNAL Pos INSERT_STRING(deflate_state *const s, const uint32_t str, uint32_t count) {
-    Pos idx, ret;
+    Pos head, idx, ret=0;
```

Comment:
Consider whitespace between operator ie `ret = 0`

Reply:
Since `ret=0` gets removed in the next commit and is a purely cosmetic change, I hope it is ok if I leave that as-is. 😄